tests/x11/yast2_snapper.pm

```diff
@@ -51,8 +51,13 @@ sub clean_and_quit() {
     if (!$ret) {
         $self->{mute_post_fail} = 1;
         record_soft_failure 'bsc#1032831';
-        $self->failure_analysis();
-        assert_screen 'root-gnome-terminal', 90;
+        # Wait another 90 seconds for the commands to finish
+        my $ret = check_screen('root-gnome-terminal', 90);
```

Comment:
The thought behind this was to wait longer and give the console a chance to close.
Multi-Tag will bring no benefit for what was intended.